test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -883,8 +883,8 @@ func getPortURL(client clientset.Interface, ns, name string, svcPort int) (strin
 	if err != nil {
 		return "", err
 	}
-	// This list of nodes must not include the master, which is marked
-	// unschedulable, since the master doesn't run kube-proxy. Without
+	// This list of nodes must not include the Control Plane, which is marked
+	// unschedulable, since the Control Plane doesn't run kube-proxy. Without
```

Comment:
```suggestion
	// This list of nodes must not include any control plane nodes, which are marked
	// unschedulable, since control plane nodes don't run kube-proxy. Without
```

Reply:
done